src/adeft/modeling/classify.py

```diff
@@ -217,22 +219,25 @@ def cv(self, texts, y, param_grid, n_jobs=1, cv=5):
         # This means metrics are calculated globally by counting all true
         # positives, false negatives, and false positives
         f1_scorer = make_scorer(f1_score, labels=self.pos_labels,
+                                pos_label=None,
```

Comment:
Input validation for `f1_score` and friends now requires explicitly setting `pos_label=None` when no `pos_labels` are desired. Before, if `average` was not `binary`, `pos_label` was completely ignored.